bcda/api/v2/api.go

```diff
@@ -18,9 +18,7 @@ import (
 var h *api.Handler
 
 func init() {
-	// TODO (BCDA-3582) - Since v2 APIs are not available (as of 2020-10-21), we're still
-	// routing requests to the v1 BFD endpoints
-	h = api.NewHandler([]string{"Patient", "Coverage"}, "/v1/fhir")
+	h = api.NewHandler([]string{"Patient"}, "/v2/fhir")
```

Comment:
What does this mean for production?  It is my understanding that BFD only deployed the `v2/Patient` resource in `prod-sbx`, but not `prod`.

Reply:
We'll leave this PR open until the v2 patient is deployed to prod

Reply:
Should we change it to `Draft PR`?

Reply:
I think it's OK to have it reviewed and approved. We just won't merge it into master until we get the greenlight from BFD.

Reply:
> What does this mean for production? It is my understanding that BFD only deployed the `v2/Patient` resource in `prod-sbx`, but not `prod`.

Followed up with the BFD team. There's no immediate plan to roll out v2 endpoints to production. For `prod`, we will not enable v2 endpoints (keep the `VERSION_2_ENDPOINT_ACTIVE` set to `false`), but enable the other env (dev, test, sbx).